internal/storage/pebbledb.go

```diff
@@ -14,6 +14,7 @@ type PebbleDB struct {
 	db *pebble.DB
 }
 
+// compile-time check: does *PebbleDB satisfy the DB interface?
```

Comment:
Don't see a strict need for this comment, as most devs are aware of such constructs